File: lib/java/src/main/java/io/github/brenbar/uapi/internal/GetMockUApiJson.java
package io.github.brenbar.uapi.internal;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.stream.Collectors;

public class GetMockUApiJson {
    public static String getMockUApiJson() {
        final var stream = Thread.currentThread().getContextClassLoader()
                .getResourceAsStream("mock-internal.uapi.json");
        return new BufferedReader(new InputStreamReader(stream)).lines().collect(Collectors.joining("\n"));
    };
}

File: tool/uapicodegen/uapicodegen/uapi/internal/schema/GetMockUApiJson.py
import os


def get_mock_uapi_json() -> str:
    file_path = os.path.join(os.path.dirname(
        __file__), "..", "..", "mock-internal.uapi.json")
    file_path = os.path.normpath(file_path)
    with open(file_path, "r") as stream:
        return "\n".join(stream.readlines())
